spec/fixtures/unit/provider/package/pkgng/pkg.info

```diff
@@ -1,7 +1,6 @@
 ca_root_nss 3.15.3.1 security/ca_root_nss
 curl 7.33.0 ftp/curl
 gnupg 2.0.22 security/gnupg
-mcollective 2.2.4 sysutils/mcollective
```

Comment:
I don't know how this fixtures file was generated, but we should confirm that this entry wasn't testing a specific codepath in this provider before we remove it.